docs/howto/how_to_leverage_scenario_editor.md

```diff
@@ -0,0 +1,130 @@
+
+Several developers wrote in requesting that our Dreamland platform should support Scenario Creation and Editing which the Apollo team now proudly presents in Apollo 5.0!
+
+```
+Please note,
+In order to use this feature, please send an email to our BD team at idg-apollo@baidu.com using your company email as it is currently available only for select partners
+```
+
+## Setting up Scenario Editor
+
+1. Login to your Dreamland account. For additional details on How to create an account, please refer to [our Dreamland Introduction Guide](https://github.com/ApolloAuto/apollo/blob/master/docs/specs/Dreamland_introduction.md) 
+
+2. Once inside the platform, the Scenario Editor can be accessed under `Scenario Management` or using the [following link](https://azure.apollo.auto/scenario-management/scenario-editor)
+
+    ![](images/se_location1.png)
+
+3. Once inside, you should see the prompt as seen in the image below, requesting you to send us an email from your company's email. As this app is in Beta testing, it is not open to all our developers.
+
+    ![](images/login_se.png)
+
+4. In 3 business days, you should receive an email confirming your account's approval and activation, after which, your screen should no longer display the prompt and the buttons would no longer be greyed out.
+
+## Using Scenario Editor
+
+Congratulations! You are now ready to use our scenario editor.
+
+1. The first step is to select a map. Currently, we offer 2 maps - Sunnyvale and San Mateo
+
+    ![](images/se_map.png)
+
+2. Once a map has been selected, you will have access to the editor pane on the right along with other tools as seen below:
+
+    ![](images/se_tools.png)
+
+
+You can navigate through the map using your arrow keys. Alternatively, you can right-click the mouse and drag it to move the map. If you are using a trackpad, you will have to two-finger click drag with two fingers.
+Lets understand each tool along with its purpose
+
+### General Action Tools
+
+The 4 General action tools can be found on the bottom right corner of the map. 
+
+1. **Zoom tool**: while you can use your trackpad to zoom in and out of the map, there exists the Zoom tool to help you zoom in and out of the map in case you do not have a trackpad ready.
+
+    ![](images/se_zoom1.png)
+
+2. **Re-center tool**: this tool allows you to locate you ego-car on the map even if you have moved away
+
+    ![](images/se_center1.png)
+
+3. **Ruler tool**: this tool allows you to measure the distance between two points. 
+
+    ![](images/se_ruler.png)  ![](images/se_distance.png)
+
+4. **Add Route tool**: you can set the ego car’s end point by clicking on the Add Route tool. Please note that the first version of the Scenario Editor does support the creation of multiple route points for the ego car.
```

Comment:
Yes included the uses of "Add route point" separately in the participants section to avoid confusion. Changed this language accordingly to reduce confusion.